app/delivery/send_to_providers.py

```diff
@@ -253,13 +252,10 @@ def get_logo_url(
     return 'https://{}.{}/{}'.format(bucket, domain, logo_file)
 
 
-def get_html_email_options(
-    notification,
-    provider,
-):
+def get_html_email_options(notification: Notification) -> Dict[str, Union[str, bool]]:
```

Comment:
nit: python 3.10 eliminates the need for this. You no longer need to import Union at all. All the details are in [PEP 604](https://www.python.org/dev/peps/pep-0604/). The PEP allows you to replace it entirely with the `|` operator.

Reply:
Updated.